src/style.css

```diff
@@ -1 +1,144 @@
-/* add styles here */
+.stars {
+  margin: 0;
+}
+
+.stars__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.classes`. Here, you're styling the `.stars` and `.stars__list` classes, which is good, but ensure that any other tag names are not styled directly in the rest of your CSS.